src/test/java/api/support/builders/FeeFineBuilder.java

```diff
@@ -0,0 +1,76 @@
+package api.support.builders;
+
+import static org.folio.circulation.support.JsonPropertyWriter.write;
+
+import java.util.UUID;
+
+import io.vertx.core.json.JsonObject;
+
+public class FeeFineBuilder extends JsonBuilder implements Builder {
+  private UUID id;
+  private UUID ownerId;
+  private String feeFineType;
+  private Double defaultAmount;
+  private String chargeNoticeId;
+  private String actionNoticeId;
+  private JsonObject metadata;
+
+  public FeeFineBuilder() {
+  }
+
+  public FeeFineBuilder(UUID id, UUID ownerId, String feeFineType, Double defaultAmount,
+                        String chargeNoticeId, String actionNoticeId,
+                        JsonObject metadata) {
```

Comment:
Please reformat this to use standard line separation and indentation rather than horizontal alignment (which is harder to maintain and wastes horizontal space)

Reply:
Done.